app/less/table.less

```diff
@@ -9,6 +9,6 @@ tbody {
     background-color:rgba(0, 0, 0, 0.2);
   }
   tr.selected {
-    background-color:rgba(0, 30, 0, 0.8);
+    background-color:rgba(129, 212, 250, 0.8);
```

Comment:
Does not improve visability but makes it worse - espacially since the link now is almost invisible. Has to be removed again.